v2/e2e/kustomization.yaml

```diff
@@ -1,15 +1,15 @@
 resources:
-- ../config/default
-- ../config/pod/coil-router.yaml
+  - ../config/default
+  - ../config/pod/coil-router.yaml
```

Comment:
```suggestion
- ../config/default
- ../config/pod/coil-router.yaml
```

nit: Other yaml files don't have spaces.

Reply:
my editor added spaces automatically.
I fixed this.

Reply:
They're still around 👀